### PR TITLE
Refactor test suite

### PR DESCRIPTION
**Pull request by wexo-team**
Alternative to #234 with PHPUnit support

### CODE REVIEW

**Comment by centerax** [comment]
any problems having the tests at test/unit?

**Comment by wexo-team** [comment]
The `dev` directory has been in Magento since 1.9.2.0, so most Magento hosting providers are already configured to protect it.

Creating a new `test` directory whould require you to add additional config to your web server in order to make it unavailable to the public.

**Comment by centerax** [comment]
environmental variable is set. not sure why it does not work now
screen shot 2017-05-17 at 5 13 49 pm

File: modman [hunk 1]
```diff
@@ -5,6 +5,7 @@ app/design/adminhtml/default/default/template/ebizmarts/mailchimp    app/design/
 app/design/frontend/base/default/layout/ebizmarts/mailchimp.xml      app/design/frontend/base/default/layout/ebizmarts/mailchimp.xml
 app/design/frontend/base/default/template/ebizmarts/mailchimp        app/design/frontend/base/default/template/ebizmarts/mailchimp
 app/etc/modules/Ebizmarts_MailChimp.xml                              app/etc/modules/Ebizmarts_MailChimp.xml
+dev/tests/ebizmarts_mailchimp/                                       dev/tests/ebizmarts_mailchimp/
```
Comment (centerax) [inline modman]: can we rename this to **dev/tests/mailchimp** ?

Comment (wexo-team) [inline modman]: Yes we can and done it is

**Pull request merged**